common/v2/features/AddAccount/components/Mnemonic.tsx

```diff
@@ -2,11 +2,13 @@ import React, { PureComponent } from 'react';
 import { connect } from 'react-redux';
 import { mnemonicToSeed, validateMnemonic } from 'bip39';
 
+import { FormData } from 'v2/features/AddAccount/types';
```

Comment:
**convention** Only import at the service level.
If the type needs to shared move it into `v2/types`.
In this case, I don't see any no reason to share `FormData`.
(see below)

Reply:
I use `FormData` as it is passed to all steps in the `AddAccountFlow` and updated in `NetworkSelectPanel`. It seemed like it was intended to be used directly instead of manually adding a prop for network.

Reply:
If not, please let me know.

Reply:
It that case it's out of scope of this PR.
Thanks for pointing it out!
(...and git blame shows me I'm the culprit...since then I prefer being more explicit then sharing the form itself.)